Makefile

```diff
@@ -59,3 +60,10 @@ clean:
 	rm template/internal/deftmpl/bindata.go
 	rm ui/bindata.go
 	cd $(FRONTEND_DIR) && $(MAKE) clean
+
+.PHONY: test
+test: common-test
+	$(ERRCHECK_BINARY) -verbose -exclude scripts/errcheck_excludes.txt -ignoretests ./...
```

Comment:
If this fails, can you output a message that mentions where to find the exceptions file?